docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.install

```diff
@@ -283,3 +283,36 @@ function _su_humsci_profile_update_fix_menuposition_sections(array $sections) {
   }
   return $changed;
 }
+
+/**
+ * Uninstall material admin theme and theme switcher module.
+ */
+function su_humsci_profile_update_8012() {
+  $config_factory = \Drupal::configFactory();
+
+  // Delete the old block configurations for the admin theme so that when the
+  // new theme is enabled, the UUIDs will still work and be the same.
```

Comment:
Thanks for the comment details.

Reply:
woops. thats actually an old comment. they arent doing that

Reply:
Damn, that seemed magical.

Reply:
well its actually not even needed. i went one approach of changing which admin theme was set. that was complicated. so i just change the base theme of this. much easier